src/poliastro/twobody/orbit.py

```diff
@@ -372,40 +373,47 @@ def from_body_ephem(cls, body, epoch=None):
 
         """
         # TODO: https://github.com/poliastro/poliastro/issues/445
-        if not epoch:
-            epoch = time.Time.now().tdb
-        elif epoch.scale != "tdb":
-            epoch = epoch.tdb
-            warn(
-                "Input time was converted to scale='tdb' with value "
-                "{}. Use Time(..., scale='tdb') instead.".format(epoch.tdb.value),
-                TimeScaleWarning,
-            )
-
-        r, v = get_body_barycentric_posvel(body.name, epoch)
+        if body.name.lower() not in solar_system_ephemeris.bodies:
```

Comment:
I guess this was not needed. You just had to apply a check for Pluto and Moon in that case!

Reply:
i think, we need this as there are several ephemeris available and some contain pluto and some may not
here is the list of ephemeris i found on astropy
- 'builtin': polynomial approximations to the orbital elements.
- 'de430' or 'de432s': short-cuts for recent JPL dynamical models.
- 'jpl': Alias for the default JPL ephemeris (currently, 'de430').
- URL: (str) The url to a SPK ephemeris in SPICE binary (.bsp) format.
- `None`: Ensure an Exception is raised without an explicit ephemeris

pls review

Reply:
here, using url we can add any ephemeris

Reply:
I believe we didn't intend to achieve this from the issue! @Juanlu001  I will seek your help.